dynamic_rest/serializers.py

```diff
@@ -301,6 +317,29 @@ def get_name(cls):
         return cls.Meta.name
 
     @classmethod
+    def get_url(self, pk=None):
+        # if associated with a registered viewset, use its URL
+        url = getattr(self, '_url', None)
+        if url:
+            # use URL key to get endpoint
+            url = reverse(url)
+        if not url:
+            # otherwise, return canonical URL for this model
+            from dynamic_rest.routers import DynamicRouter
+            url = DynamicRouter.get_canonical_path(
```

Comment:
:+1: to the safeguard, see my other comment about `get_canonical_path` vs `reverse`